src/functions-and-arrays.js

```diff
@@ -52,14 +139,38 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(array) {
+  let uniquifyArray = [];
+  if ( array.length === 0 ) {
+    return null;
+  } else {
+  for ( let i = 0; i < array.length; i++ ) {
+    const item = array[i];
+    if ( !uniquifyArray.includes(item) ) {
+      uniquifyArray.push(item)
+    }
+  }
+}
+return uniquifyArray
+}
+//console.log(uniquifyArray(wordsUnique))
+ 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(words, search) {
+
+  if ( words.length === 0 ) {
+    return null;
+  } else if ( words.includes(search) ) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
```suggestion
  } else {
    return words.includes(search);
  }
```

El propio metodo includes retorna un true o un false, asi no tienes que hacerte el If!! :D
La docu la tienes aqui!
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes